content/integrations/aws.md

```diff
@@ -105,7 +106,10 @@ Note: The GovCloud and China regions do not currently support IAM role delegatio
                 "sns:List*",
                 "sns:Publish",
                 "sqs:ListQueues",
-                "support:*"
+                "support:*",
```

Comment:
http://docs.aws.amazon.com/IAM/latest/UserGuide/list_trustedadvisor.html unfortunately it requires *